src/components/ActNowCard/PetitionCard.js

```diff
@@ -18,11 +20,17 @@ function PetitionCard({ closeCard, promiseActNow, ...props }) {
 
   const { petitionJoin, petitionTitle: petitionStart } = props;
   const [open, setOpen] = useState(false);
+  const { session } = useSession();
```

Comment:
I don't think the prop is called `session`.

Reply:
its `data` @nerdlet

Reply:
Yap, @KobbyMmo @kilemensi  it's data